src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.Parser.cs

```diff
@@ -580,7 +616,7 @@ private ObjectSpec CreateObjectSpec(TypeParseInfo typeParseInfo)
                     ImmutableArray<ISymbol> members = current.GetMembers();
                     foreach (ISymbol member in members)
                     {
-                        if (member is IPropertySymbol { IsIndexer: false, IsImplicitlyDeclared: false } property)
+                        if (member is IPropertySymbol { IsIndexer: false, IsImplicitlyDeclared: false } property && !IsUnsupportedType(property.Type))
```

Comment:
Do we need to do anything different for constructor initialization? I suppose it's ok just to omit the members and then expect them to be missing from constructor- when present the error message might be confusing. Not sure if there's anything else rough about that scenario

Reply:
That is a good point. I'll try fix the constructor case. I cannot think of any other scenario that needs to be handled here.

Reply:
I did the suggested constructor parameters check.